### PR TITLE
Added support for the "working directory" option to tar, -C.

### PR DESCRIPTION
**Pull request by terjekv**
- Modified checking of paths to take working path into account.
  - Modified the example to include the appropriate key.

### CODE REVIEW

File: feather [hunk 1]
```diff
@@ -183,9 +183,15 @@ class pr_tarsnap(object):
             path = self.backups[key]['path']
             if isinstance(path, list):
                 for item in path:
-                    self.check_valid_path(item)
+                    path_to_check = item
+                    if 'workingpath' in self.backups[key]:
+                        path_to_check = self.backups[key]['workingpath'] + '/' + item
```
Comment (danrue) [inline feather]: Can you change this to use os.path.join?

**Comment by danrue** [comment]
Thanks for the change. It looks great. Could you change the path join to use os.path.join, and also update README with the latest version of feather.yaml.dist? It is just copied in there.

**Comment by terjekv** [comment]
Good points, I'll fix this and resubmit.

**Comment by danrue** [comment]
Thanks @terjekv! It's a great change. :+1:

**Pull request merged**